packages/paste-core/components/toast/src/types.ts

```diff
@@ -0,0 +1,20 @@
+import * as PropTypes from 'prop-types';
+import {ValueOf} from '@twilio-paste/types';
+import {ToastVariantObject} from './constants';
+
+export type ToastVariants = ValueOf<typeof ToastVariantObject>;
+
+export interface ToastProps {
+  children: NonNullable<React.ReactNode>;
+  onDismiss?: () => void;
+  role?: string;
```

Comment:
I don't think you mean to have a role as part of the interface here. You don't actually allow folks to set it

Reply:
Yep. Missed that when I removed the different roles.